modules/test/playwright/tests/frontend-data-set-views-web/visualizationModes.spec.ts

```diff
@@ -374,6 +374,45 @@ test.describe('Visualization Modes in Data Set Manager', () => {
 			await visualizationModesPage.cancelAddFieldsModal();
 		});
 	});
+
+	test('Configure table visualization mode with scalar array fields @LPD-11769', async ({
+		visualizationModesPage,
+	}) => {
+		const SAMPLE_SCALAR_ARRAY_FIELD = 'keywords';
```

Comment:
good! hope this field is considered API in the objects side of things so that we don't have to change this